### PR TITLE
Issue 1750 - programmatic storage deal making

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses https://github.com/filecoin-project/filecoin-docs/issues/1759

Supersedes https://github.com/filecoin-project/filecoin-docs/pull/1779. Uses issue description https://github.com/filecoin-project/filecoin-docs/pull/1779#issue-1620642485 as basis for the content in this PR.

### CODE REVIEW

**Comment by ElPaisano** [comment]
@johnnymatthews @trruckerfling can you take a look at this PR? It recreates and replaces https://github.com/filecoin-project/filecoin-docs/pull/1779.

A few things:
- Wasn't exactly sure where this content should go in the new info architecture, so I added it to a new file `content/en/smart-contracts/developing-contracts/fvm-storage-deals/index.md`. Should it go here or somewhere else?
- In the original copy https://github.com/filecoin-project/filecoin-docs/pull/1779#issue-1620642485, the existence of tutorials for each method is implied. Where are those tutorials? Are they currently being written and can be linked to later? Or do they need to be added into this PR.
- @trruckerfling I did some rewriting / rearranging of your original content here, please LMK if I missed something or am explaining something incorrectly.

**Comment by ElPaisano** [comment]
Putting this on hold until the two tutorials that need to be linked to are available

**Comment by ElPaisano** [comment]
@trruckerfling wanted to follow up with you on this one. Are the tutorials that need to be linked here available somewhere? Or am I misunderstanding that that's what's missing here? Or do I need to document them 😅 ? Apologies for the confusion here

File: content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Using the programmatic storage building blocks provided by the FVM, developers can build: 
+- [DataDAOs](https://rfs.fvm.dev/#482335d5b27543b3b0d5e81b995b719d), which implement a main client contract that adds CIDs to store new data through a voting mechanism.
+- [Perpetual storage](https://rfs.fvm.dev/#2934f09e0d8c4280a6d9a5c08ffdeee1), which implements client contracts that recycle CIDs from expiring deals into their authorization sets, thereby allowing data to be stored perpetually.
+
+{{< alert >}}
+For more FVM use case ideas, see the [Request for Startups](https://rfs.fvm.dev/).
+{{< /alert >}}
+
+Descriptions of each method, as well as links to tutorials describing the methods and best practices for programmatic storage deal making, are provided below.
```
Comment (ElPaisano) [inline content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md:25]:
```suggestion
Descriptions of each method are provided below.
```

File: content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+- Data preparation is required before initiating the deal proposal from the client contract, as the data preparation outputs several pieces of information that is required for the deal proposal - specifically, a [CAR file](https://spec.filecoin.io/systems/filecoin_files/piece/#:~:text=A%20CAR%20file%20is%20an,provider%20as%20we%20discuss%20later.) link, the CAR file size, the data file size and the [data CID](https://spec.filecoin.io/systems/filecoin_files/piece/). Manual and automated options for the data preparation will be shown in the client contract tutorial. 
+
+- Currently, the client contract only works with SPs running the latest [Boost](https://boost.filecoin.io/) release, [v1.6.0](https://github.com/filecoin-project/boost/releases/tag/v1.6.0) AND who have enabled the FVM dealmaking deature. SPs who meet these requirements will be listening on-chain for client contract deal proposals events to accept. 
+
+For more information, see the tutorial describing this method.
```
Comment (ElPaisano) [inline content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md:35]:
```suggestion
```

File: content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+In this method, an FEVM smart contract, referred to as a _deal bounty contract_, represents the storage proposal from the client, **but does not manage the deal making**. Instead, the contract incentivizes a third party, referred to as a _bounty hunter_, to pass the deal proposal on to the SP and secure the deal on behalf of the client. 
+
+The deal bounty contract:
+
+- Consists of a list of data CIDs pointing to the data to be stored, which the bounty hunter picks up.
+- Will hold the deal bounty in escrow on behalf of the client.
+- Will pay out the deal bounty to the bounty hunter once a deal is made and the deal ID is returned and validated.
+- Will check with the Filecoin built-in storage market actor to confirm whether the supplied deal ID is activated and stores the claimed data.
+
+As with the [client contract method](#using-a-client-contract), data has to be prepared in order to add the data CID to the contract. For more information, see the tutorial describing this method.
```
Comment (ElPaisano) [inline content/en/smart-contracts/fundamentals/fvm-storage-deals/index.md:48]:
```suggestion
As with the [client contract method](#using-a-client-contract), data has to be prepared in order to add the data CID to the contract.
```

**Comment by johnnymatthews** [comment]
This PR is ancient, and I'm pretty sure the content has already been addressed elsewhere. I'm gonna close this.

**Pull request closed without merging**